src/ImmersedBoundaries/grid_fitted_bottom.jl

```diff
@@ -63,34 +63,35 @@ function Base.show(io::IO, ib::GridFittedBottom)
 end
 
 """
-    ImmersedBoundaryGrid(grid, ib::GridFittedBottom)
+    ImmersedBoundaryGrid(grid, immersed_boundary::GridFittedBottom)
 
-Return a grid with `GridFittedBottom` immersed boundary.
+Return a grid with `GridFittedBottom` immersed boundary (`immersed_boundary`).
 
-Computes ib.bottom_height and wraps in an array.
+Computes `immersed_boundary.bottom_height` and wraps it in a Field.
 """
-function ImmersedBoundaryGrid(grid, ib::GridFittedBottom)
+function ImmersedBoundaryGrid(grid, immersed_boundary::GridFittedBottom)
     bottom_field = Field{Center, Center, Nothing}(grid)
-    set!(bottom_field, ib.bottom_height)
+    set!(bottom_field, immersed_boundary.bottom_height)
     fill_halo_regions!(bottom_field)
-    new_ib = GridFittedBottom(bottom_field, ib.immersed_condition)
+    new_immersed_boundary = GridFittedBottom(bottom_field, immersed_boundary.immersed_condition)
     TX, TY, TZ = topology(grid)
-    return ImmersedBoundaryGrid{TX, TY, TZ}(grid, new_ib)
+    return ImmersedBoundaryGrid{TX, TY, TZ}(grid, new_immersed_boundary)
 end
```

Comment:
I kinda prefer it how it was before. In my opinion "verbosity" is not required when everything is self-contained and clear as in this function (`ib` here already clearly comes in as a `GridFittedBottom`)

Reply:
okie!